lib/fastlane_core/devices.rb

```diff
@@ -1,14 +1,19 @@
 require 'open3'
 
 module FastlaneCore
-  class Simulator
+  class Devices
```

Comment:
Yea i really struggled for a good name for this too, i think `DeviceManager ` may be the best description since it does things like reset simulators and such.

Let me know if you come up with any good idea for a name otherwise i will change it to `DeviceManager` if thats okay with you? :confused: